modules/mobile/app/models/mobile/v0/appointment.rb

```diff
@@ -68,7 +68,7 @@ class Appointment < Common::Resource
       attribute :best_time_to_call, Types::Array.optional
       attribute :friendly_location_name, Types::String.optional
 
-      def self.convert_non_prod_id!(id)
+      def self.convert_from_non_prod_id!(id)
```

Comment:
I think we should add a comment explaining why this is necessary and what "non_prod_id" means. I kinda wonder if "non_prod_id" is even accurate. Alastair told me that the data was copied from production. So I think it might actually be more accurate as "convert_from_prod_id". But I'm not sure enough to ask you to make the change 😅

Reply:
I'm hesitant to make that switch since Lighthouse told me that they've never seen 9xx ids for facility ids. I'm assuming they were referring to production as well. If we can get another person to verify the nature of these ids than I'd be open to it.